src/components/ConditionFieldset.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+import { defaultTheme } from "../theme";
+import { Condition } from "../utils/CarAdapter.util";
+import { Title, Validation } from "./PublishCarForm.component";
+
+export const ConditionFieldset = ({ error, register, validation }: any) => {
```

Comment:
I recommend you to read a little more about the hook form context https://react-hook-form.com/advanced-usage/#FormProviderPerformance so you know about another way to share the form methods between components